x/interchainstaking/keeper/receipt.go

```diff
@@ -248,6 +248,9 @@ func (k *Keeper) SubmitTx(ctx sdk.Context, msgs []sdk.Msg, account *types.ICAAcc
 		return sdkioerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, "module does not own channel capability")
 	}
 
+	// TO DO: discuss on adding this because we are receiving err capability not found from scoppedICAController
+	k.ICAControllerKeeper.ClaimCapability(ctx, chanCap, host.ChannelCapabilityPath(portID, channelID)) //nolint:errcheck
```

Comment:
remove nolint?

Reply:
@sontrinh16 @ThanhNhann I think we need more context this addition

Reply:
Yeah, I posted it here https://notionalgroup.slack.com/archives/C03H7U50UKY/p1687412790885519

Reply:
And more here https://github.com/ingenuity-build/quicksilver/pull/420#discussion_r1208978367